README.md

```diff
@@ -44,6 +46,15 @@ permitted_domains: 'hashrocket.com|hshrckt.com'
 permitted_emails: 'friend@whitelist.com'
 ```
 
+##### To get admin route working on localhost:
+
+```
+1. Navigate to https://console.developers.google.com/
```

Comment:
@jwworth, I included the change you requested. However, also added a reminder that users change the google client id and google secret id. 

You can delete the line if unwanted. But I think that component is not going to change and it gives the programmer affirmation that they correctly executed the Oauth instructions.